readme.md

```diff
@@ -1,163 +0,0 @@
-# electron-packager
```

Comment:
I tried having the symlinks be the other way, but then GitHub Pages wouldn’t build.